pkg/acsengine/const.go

```diff
@@ -170,31 +86,10 @@ const (
 	DefaultMasterEtcdServerPort = 2380
 	// DefaultMasterEtcdClientPort is the default etcd client port for Kubernetes master nodes
 	DefaultMasterEtcdClientPort = 2379
-	// DefaultKubeletEventQPS is 0, see --event-qps at https://kubernetes.io/docs/reference/generated/kubelet/
-	DefaultKubeletEventQPS = "0"
-	// DefaultKubeletCadvisorPort is 0, see --cadvisor-port at https://kubernetes.io/docs/reference/generated/kubelet/
-	DefaultKubeletCadvisorPort = "0"
-	// DefaultJumpboxDiskSize specifies the default size for private cluster jumpbox OS disk in GB
-	DefaultJumpboxDiskSize = 30
-	// DefaultJumpboxUsername specifies the default admin username for the private cluster jumpbox
-	DefaultJumpboxUsername = "azureuser"
-	// DefaultKubeletPodMaxPIDs specifies the default max pid authorized by pods
-	DefaultKubeletPodMaxPIDs = 100
-	// DefaultKubernetesAgentSubnetVMSS specifies the default subnet for agents when master is VMSS
-	DefaultKubernetesAgentSubnetVMSS = "10.248.0.0/13"
 	// DefaultUserAssignedID specifies the default name for the user assigned identity
 	DefaultUserAssignedID = "acsenginetestid"
 )
 
-const (
-	// DCOSMaster represents the master node type
```

Comment:
So the idea behind deleting these consts and not moving them elsewhere is that they are unused?

Should we delete the `DCOSNodeType` type struct as well?